src/components/Navbar.js

```diff
@@ -31,8 +37,33 @@ export default function Navbar() {
               className={cx(styles.inputField, styles.widthFixed)}
             />
           </li>
+
+          <li>
+          <Checkbox
+                label="Snap to grid"
+                className={cx(styles.checkGrid)}
+              />
+          </li>
+          <li>
+          <Checkbox
+                label="Ratio"
+                className={cx(styles.checkRatio)}
+              />
+          </li>
+          <li>
+          <TextField
+              name="canvasColor"
+              hintText="#ffffff"
+              className={cx(styles.inputField, styles.widthFixed)}
+            />
+        </li>
+          <li>
+           <RaisedButton className={cx('btnEnterGrid', styles.btnEnterGrid)} label="Enter grid" secondary={true} />
```

Comment:
This is the element from the old index.html

`<a class="waves-effect waves-light btn btnEnterGrid">Enter grid</a>`

But we don't use materialize.css anymore so we don't need these other classes and styles to merge. I just added new style for it in navbar.css , no sure If that is okay.